From 3d99e862691bb7053ba9e1537a971971976fde13 Mon Sep 17 00:00:00 2001
From: Stefano Babic <sbabic@denx.de>
Date: Fri, 13 Nov 2015 10:49:01 +0100
Subject: [PATCH 3/3] imx6: fix warnings due to switch to distro environment

wandboard and mx6cuboxi have warnings because BOOT_DELAY
is defined twice.

Signed-off-by: Stefano Babic <sbabic@denx.de>
CC: Vagrant Cascadian <vagrant@debian.org>
CC: Otavio Salvador <otavio@ossystems.com.br>
CC: Fabio Estevam <fabio.estevam@freescale.com>
---
 include/configs/mx6cuboxi.h | 2 +-
 include/configs/wandboard.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/mx6cuboxi.h b/include/configs/mx6cuboxi.h
index 0f3b992..4feb121 100644
--- a/include/configs/mx6cuboxi.h
+++ b/include/configs/mx6cuboxi.h
@@ -8,6 +8,7 @@
 #ifndef __MX6CUBOXI_CONFIG_H
 #define __MX6CUBOXI_CONFIG_H
 
+#include <config_distro_defaults.h>
 #include "mx6_common.h"
 
 #define CONFIG_SPL_LIBCOMMON_SUPPORT
@@ -137,7 +138,6 @@
 	func(PXE, pxe, na) \
 	func(DHCP, dhcp, na)
 
-#include <config_distro_defaults.h>
 #include <config_distro_bootcmd.h>
 
 #else
diff --git a/include/configs/wandboard.h b/include/configs/wandboard.h
index 6408b5e..4c8e9e9 100644
--- a/include/configs/wandboard.h
+++ b/include/configs/wandboard.h
@@ -9,6 +9,7 @@
 #ifndef __CONFIG_H
 #define __CONFIG_H
 
+#include <config_distro_defaults.h>
 #include "mx6_common.h"
 
 #define CONFIG_SPL_LIBCOMMON_SUPPORT
@@ -166,7 +167,6 @@
 	   "run findfdt; " \
 	   "run distro_bootcmd"
 
-#include <config_distro_defaults.h>
 #include <config_distro_bootcmd.h>
 
 /* Physical Memory Map */
-- 
2.1.4

